apps/meteor/ee/app/livechat-enterprise/server/lib/OnHoldHelper.ts

```diff
@@ -0,0 +1,105 @@
+import { Meteor } from 'meteor/meteor';
+import type { IMessage, IOmnichannelRoom, IUser } from '@rocket.chat/core-typings';
+import { isOmnichannelRoom } from '@rocket.chat/core-typings';
+import { LivechatRooms, Subscriptions, Messages, LivechatInquiry } from '@rocket.chat/models';
+
+import { RoutingManager } from '../../../../../app/livechat/server/lib/RoutingManager';
+import { callbacks } from '../../../../../lib/callbacks';
+import { ohLogger as logger } from './logger';
+
+class OnHoldHelperClass {
```

Comment:
Let's move this to be a service, under ee/server/services

Reply:
This could be a "omnichannel-ee" service that will hold ee features for omni 🤗

Reply:
Let me try this. I wanted to do this before but since this method had some meteor code in the form of callbacks, I went with this approach.

Reply:
Moved it to service and everything is working fine :partying_face: